openmicroscopy/openmicroscopy
# Api images

# What this PR does

Adds support for ```/images/``` and ```/images/:image_id/```.

Added integration tests of urls below.
Also added gateway test for ```getObject()``` with 'load_pixels' and 'load_channels' and one for 'orphaned'.

# Testing this PR

- Check tests green.
- Browse from /api/ -> version -> images_url to list all images at e.g. ```/api/v0.1/m/images/```
- Filter by dataset ```/images/?dataset=:id``` or ```datasets/:dataset_id/images/```
- List orphaned images ```/images/?orphaned=true```
- For all above urls, pixels should be loaded for images. json contains "Pixels"
- Show a single image ```/images/:image-id/```
- This should also load Channels data.

# Related reading

https://github.com/openmicroscopy/design/issues/69
